src/tasks/tasks.py

```diff
@@ -38,6 +38,7 @@
 
 from typing import Iterable, Sequence, List, Dict, Any, Type
 
+BERT_INDEXER_NAME = "bert_wpm_pretokenized"
```

Comment:
I slightly prefer having the constant, but I'll revert (mainly because this was I agree this should go in `bert/utils.py` and it was leading to circular dependencies...)